src/SlmQueue/Strategy/ProcessQueueStrategy.php

```diff
@@ -40,6 +40,8 @@ public function onJobPop(WorkerEvent $e)
         $worker       = $e->getTarget();
         $eventManager = $worker->getEventManager();
 
+        $e->setJob($job);
```

Comment:
At this point you're not sure that you have a JobInterface instance (see the check next line), so this call may raise an exception.